app.js

```diff
@@ -49,7 +49,8 @@ const path = require('path');
 try {
 	require.resolve('sockjs');
 } catch (e) {
-	throw new Error("Dependencies unmet; run npm install");
+	console.error('Dependencies are unmet; run npm install before launching Pokemon Showdown again.');
+	process.exit(1);
```

Comment:
@xfix would be able to explain better, since the problems with simply throwing an error instead of exiting don't happen on my system. Maybe it's dependent on OS or the version of Node running